front/components/assistant/conversation/AssistantQuickEditionMenu.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  ClipboardIcon,
+  DropdownMenu,
+  Icon,
+  ListAddIcon,
+  ListRemoveIcon,
+  MoreIcon,
+  PencilSquareIcon,
+} from "@dust-tt/sparkle";
+import type { AgentUserListStatus, WorkspaceType } from "@dust-tt/types";
+import {
+  isAgentConfigurationInWorkspace,
+  isAgentConfigurationPublished,
+  isBuilder,
+  isGlobalAgentConfiguration,
+} from "@dust-tt/types";
+import { useContext, useState } from "react";
+
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { performAgentUserListStatusUpdate } from "@app/lib/client/dust_api";
+import { useAgentConfiguration } from "@app/lib/swr";
+
+interface AssistantQuickEditionMenuProps {
+  agentConfigurationSId: string;
+  owner: WorkspaceType;
+}
+
+export function AssistantQuickEditionMenu({
+  // The `agentConfiguration` cannot be used directly as it isn't dynamically
+  // updated upon user list mutations. This limitation stems from its
+  // propagation method from <ConversationMessage>.
+  agentConfigurationSId,
```

Comment:
To be confirmed, but I think spolu told me to call that agentConfigId (not sId) => in the upper front layer, there are no model ids, all ids should be sIds (cf just below here 'workspaceId')

Reply:
Ok good to know, let me update.